presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -190,7 +190,7 @@ You may also wish to set the following properties:
   Specifies the port for the JMX RMI server. Presto exports many metrics
   that are useful for monitoring via JMX.
 
-See also :doc:`/admin/resource-groups`.
+See also  :doc:`/security`, :doc:`/admin/resource-groups`.
```

Comment:
What do you think about expanding this to be clear that the above is a minimal set?
```
The above configuration properties are a minimal set to help you get started.
Please see :doc:`/admin` and :doc:`/security` for a more comprehensive list.
In particular, see :doc:`/admin/resource-groups` for configuring queuing policies.

Reply:
Sounds good!